### ISSUE TITLE
GraspGrid doesn't handle elliptical truncation

### ISSUE THREAD

**Issue by PaulKGrimes** [PaulKGrimes/GraspFile#1]
GraspGrid objects can't read GRASP Field Data in Grids files with elliptical truncation.

For details of the formatting for elliptically truncated grids, see TICRA Tools 19.1.0 manual page 2361.

**Comment by PaulKGrimes** [comment]
There is a small bug in GraspGrid.py -> GraspField.readGraspField() line 74.
Previous i_e was being set to be equal to the second value on the line specifying the position and number of entries - it should be the sum of the start position and the number of entries.

```
           if self.kLimit == 1:
                line = f.readline().split()
                i_s = int(line[0])-1
                i_e = i_s + int(line[1])
```

**Comment by PaulKGrimes** [comment]
Fixed by cbb418e5523123650db0de104a190441425d2dc8 and merged into master with c3deb4aabc605ad35b65a72ed6c2d6384b38dd18
